sig-list.md

```diff
@@ -0,0 +1,45 @@
+
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see github.com/kubevirt/community/tree/main/generator/cmd/sigs/README.md
+--->
+
+Most community activity is organized into Special Interest Groups (SIGs),
+Working Groups, and the community meeting.
+
+SIGs follow these [guidelines](GOVERNANCE.md) although each of these groups may operate a little differently
+depending on their needs and workflow.
+
+Each group's material is in its subdirectory in this project.
+
+
+### User Groups
+
+| Name | Label | Chairs | Contact | Meetings |
+|------|-------|--------|---------|----------|
+|[KubeVirt Community](kubevirt-community/charter.md) |[kubevirt-community](https://github.com/kubevirt/kubevirt/labels/kubevirt-community)| |[Slack](virtualization)<br/> [Mailing List](https://groups.google.com/forum/#!forum/kubevirt-dev) |* Regular KubeVirt Community Meeting: [ Wednesday at 15:00 UTC (weekly) ](https://zoom.us/j/92221936273)<br/>  |
+
+
+
+
+### Special Interest Groups
+
+| Name | Label | Chairs | Contact | Meetings |
+|------|-------|--------|---------|----------|
+|[sig-compute](sig-compute/charter.md) |[sig/compute](https://github.com/kubevirt/kubevirt/labels/sig/compute)|* [Itamar Holder](https://github.com/iholder101), Red Hat<br/>* [Jed Lejosne](https://github.com/jean-edouard), Red Hat<br/> |[Slack](kubevirt-dev)<br/> [Mailing List](https://groups.google.com/forum/#!forum/kubevirt-dev) | |
+|[documentation](sig-documentation/charter.md) |[](https://github.com/kubevirt/kubevirt/labels/)| | | |
+|[storage](sig-storage/charter.md) |[](https://github.com/kubevirt/kubevirt/labels/)| | | |
+|[testing](sig-testing/charter.md) |[](https://github.com/kubevirt/kubevirt/labels/)| | | |
+|[network](sig-network/charter.md) |[](https://github.com/kubevirt/kubevirt/labels/)| | | |
+|[KubeVirt Perf and Scale SIG](sig-scale/charter.md) |[kubevirt-community](https://github.com/kubevirt/kubevirt/labels/kubevirt-community)|* [Ryan Hallisey](https://github.com/rthallisey), Nvidia<br/>* [Alay Patel](https://github.com/alaypatel07), Nvidia<br/> |[Slack](kubevirt-dev)<br/> [Mailing List](https://groups.google.com/forum/#!forum/kubevirt-dev) |* KubeVirt Perf and Scale SIG Meeting: [ Thursday at 14:00 UTC (bimonthy) ](https://zoom.us/j/3205945033)<br/>  |
```

Comment:
Not so much a problem but a possible improvement: The `*`s aren't being marked up as expected, presumably because they're in a table. As such, we may want to use the html ` <ol><li>$Chair1</li><li>$Chair2</li></ol>` tags in order to have numbered items- if that is easy to add to your script. (Or if it's easier logic we could specify `1. $Chair1<br> 2. $Chair2`)
Similarly we may also want to add html bullet tags `<ul><li>Meeting1</li><li>Meeting2</li></ul> for the "Contact" and "Meeting" items to better differentiate them.

Reply:
Good points!